.changeset/large-stingrays-fry.md

```diff
@@ -0,0 +1,14 @@
+---
+'astro': minor
+---
+
+Add an experimental Dev Overlay. This overlay allows you to inspect your page islands, see helpful audits on performance and accessibility, and more.
```

Comment:
```suggestion

Dev Overlay (experimental)

Provides a new dev tool for your browser preview that allows you to inspect your page islands, see helpful audits on performance and accessibility, and more.

Also provides a new Dev Tool Overlay Plugin API to allow you to add new features and third-party integrations to the existing dev overlay.
```
Since this is a minor, and will need a blog post blurb generated, what about something a tiny bit beefier like this? Make it accurate/what you want to say, but I think this kind of structure could work?